run.sh

```diff
@@ -102,6 +102,7 @@ es_node_start=" --network devnet \
   --l1.beacon-based-time 1706684472 \
   --l1.beacon-based-slot 4245906 \
   --download.thread 32 \
+  --state.upload.url http://65.109.63.154:8080 \
```

Comment:
Do we want to bind a subdomain to the server to make changing the IP more convenient

Reply:
http://metrics.ethstorage.io has been mapped to 65.109.63.154